src/main/java/gyro/google/compute/HealthCheckResource.java

```diff
@@ -0,0 +1,310 @@
+package gyro.google.compute;
+
+import java.io.IOException;
+import java.util.Set;
+
+import com.google.api.client.googleapis.json.GoogleJsonResponseException;
+import com.google.api.client.util.Data;
+import com.google.api.services.compute.Compute;
+import com.google.api.services.compute.model.HTTP2HealthCheck;
+import com.google.api.services.compute.model.HTTPHealthCheck;
+import com.google.api.services.compute.model.HTTPSHealthCheck;
+import com.google.api.services.compute.model.HealthCheck;
+import com.google.api.services.compute.model.Operation;
+import com.google.api.services.compute.model.SSLHealthCheck;
+import com.google.api.services.compute.model.TCPHealthCheck;
+import gyro.core.GyroException;
+import gyro.core.GyroUI;
+import gyro.core.Type;
+import gyro.core.resource.Resource;
+import gyro.core.scope.State;
+
+/**
+ * Creates a global health check resource.
```

Comment:
```suggestion
 * Creates a global health check.
```

Resource is a term we use, but when a person is reading the docs, they should under see the model name, also following convention of how we do things.